packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -280,20 +271,20 @@ export abstract class TestNode extends vscode.TreeItem {
     if (outcome === 'Pass') {
```

Comment:
Asking because you're in here anyway; is it worth extrapolating all the outcome strings into variables?

Reply:
good idea..on it